### DISCUSSION TITLE
Subscription Not Found

### DISCUSSION THREAD

**Discussion by kiwidyne** [rcl-ssl/docs#20] [General]
I have an active subscription to "RCL Client for Let's Encrypt". It has been active since June 17th, 2021. I have been paying the monthly fee after the first free month.

On the Azure Portal it has a link that says "Open SaaS Account on publisher's site". When I click it, I get an error message that says **"ERROR - A subscription associated with this id is already active."**

OK, that's kind of annoying. So, I'll just navigate to https://rclapp.azurewebsites.net/ .

I get a page that says "Subscription Not Found" and "Please subscribe before you use this feature." even though I'm logged in with the same account that it just said that there was already an active subscription associated with. What is going on?

Meanwhile, my client's website is throwing warning messages because the certificate has expired and wasn't automatically renewed.

Help, please.

**Comment by rcladmin** [comment]
Can you email me the username of your account. Send the email to support@rclapp.com

**Comment by kiwidyne** [comment]
OK, it turns out, Microsoft has *two* profiles associated with my login ID ("work/IT" and "personal"), and was feeding RCL the wrong one. Great. So, not my fault, and not your fault.

So, I created an administrator account, and was able to log in and get the cert renewed. Sorry for the fuss.

But if anyone sees this and has the same problem, start digging into the details for your account. You may very well have multiple profiles associated with it.

**Comment by rcladmin** [comment]
OK, please feel free to contact me if you run in to any problems in the future.

**Reply by kiwidyne** [reply]
Thank you. Sorry for the trouble. Sometimes Microsoft's account management is a bit "scattered".

**Reply by rcladmin** [reply]
I know, I am still having trouble understanding personal and work accounts, it is quite confusing. It also complicates the RCL Let's Encrypt app, I have to support work accounts and personal accounts in a very confusing way.

Anyway, thanks for your observation and comments. It may help someone in the future. Bye.
